docs/developers/contract-api-limits.md

```diff
@@ -0,0 +1,27 @@
+---
+id: contract-api-limits
+title: MultiversX Smart Contracts API limits
+---
+
+[comment]: # (mx-abstract)
+
+## MultiversX Smart Contracts API limits
+
+Starting with the release v1.4.8, we have added maximum API limits that a Smart Contract can make towards the
+node's API. These limits are set in the gas schedule files in the `MaxPerTransaction` section. For example, this 
+gas schedule file https://github.com/multiversx/mx-chain-mainnet-config/blob/master/gasSchedules/gasScheduleV7.toml has the following limits:
+
+```toml
+[MaxPerTransaction]
+    MaxBuiltInCallsPerTx = 100
+    MaxNumberOfTransfersPerTx = 250
+    MaxNumberOfTrieReadsPerTx = 1500
+```
+
+which translates to:
+* each transaction can make a maximum 100 built-in functions calls, such as "get last nonce", "get last randomness", "ESDT unpause", "ESDT NFT create" and so on;
```

Comment:
by transaction, you mean a full transaction cycle (including scrs, callbacks and so on) or a single SCR ?

Reply:
done

Reply:
nothing has changed, why 'done' ?